libdbus-sys/build.rs

```diff
@@ -1,8 +1,320 @@
 extern crate pkg_config;
+use std::error::Error;
+
+// Constants and gated use statements
+#[cfg(feature = "vendored")]
+mod consts {
```

Comment:
There are a lot of repeated #[cfg(feature = "vendored")] in this file, how about splitting it to a separate file and include it somehow, e g through `#[path = "buildvendored.rs"] mod buildvendored;`

Reply:
Good idea! Moved everything over to `build_vendored.rs` in cf25d34a9625a8b3df82b7d2686a428484a41afa.